### PR TITLE
Fixes #18293: 

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/18293

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: bootstrap-promises/failsafe.cf [hunk 1]
```diff
@@ -139,14 +142,15 @@ body copy_from remote(server, path)
...
         };
-        trustkey    => "true";
         source      => "${path}";
         compare     => "digest";
         preserve    => "false";
         verify      => "true";
         purge       => "true";
         copy_backup => "false";
         portnumber  => "5309";
+      trust_key::
```
Comment (ncharles) [inline bootstrap-promises/failsafe.cf:152]:
```suggestion
      trust_key.!root_server::
```

Comment (amousset) [reply bootstrap-promises/failsafe.cf:152]: Why not! (root server is normally never in bootstrap policies but just in case)

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Comment by amousset** [comment]
I'll replace this with something based on a file containing the policy server's key hash

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
This one needs a manual merge, but is it still a wip ?

**Review by ncharles — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**